h/storage.py

```diff
@@ -147,6 +147,8 @@ def create_annotation(request, data, group_service):
     request.db.add(annotation)
     request.db.flush()
 
+    request.db.add(models.EsSyncTask(annotation_id=annotation.id))
+
```

Comment:
Might want to move this to the `SearchIndexService` as `queue_for_indexing(annotation_id, priority)` or something. That way we can consolidate any smarts about doing it there.

This also means all roads start to lead to the same code, rather than secret connections being made in the background.

Reply:
Yeah, that's easily enough done